pretyping/recordops.ml

```diff
@@ -19,7 +19,7 @@ open CErrors
 open Util
 open Pp
 open Names
-open Globnames
+open GlobRef
```

Comment:
Nit, but `GlobRef`, as well as other modules in `Names` is not supposed to be globally opened.

Reply:
In principle I agree, but the name of the constructors inside `GlobRef` already contain `Ref` making them non ambiguous. I'd rather prefer `GlobRef.(Const|Construct|Inductive|Var)`and in that case conflicts could arise and hence `open` would be a bad idea.

Reply:
IMO it is a matter of pollution of the "global" namespace, I agree we should rename the constructors, but that's a lot of work; anyways this is just a nit, but I think this open introduces some noise in the diff.

Reply:
> I think this open introduces some noise in the diff

I agree, but it also make the file a bit more readable (at least in my opinion, hence my change).

Reply:
Using type based disambiguation could also make files more readable, without using open.
We would need to stop enabling warning 40.

Reply:
After another look the noise is IMO too much, almost 50% of the diffs lines in that file are spurious, so I suggest we revert that change.

Reply:
I've removed the open